panther_description/config/WH02.yaml

```diff
@@ -6,3 +6,7 @@ inertia_y_offset: 0.0
 mesh_package: panther_description
 folder_path: meshes/WH02
 kinematics: mecanum
+
+velocity_x_stderr: 3.2e-3     # Selected intuitively, the need for thorough testing
+velocity_y_stderr: 1.4e-3     # Selected intuitively, the need for thorough testing
+velocity_yaw_stderr: 8.5e-3   # Selected intuitively, the need for thorough testing
```

Comment:
These are mecanum wheels, they shouldn't have the same values. It should be probably copied to WH04 instead. Also change to comment to something like `selected based on WH01 results`